src/modules/counter-comments.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
- [ ] Please don't disable eslint. Kİndly open it and fix the error.

Reply:
@ahmetbozaci eslint disable removed and fixed. 👍🏼